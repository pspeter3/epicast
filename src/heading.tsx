import * as React from "react";
import Typography from "@material-ui/core/typography";

export const Heading: React.SFC<{}> = () => {
    return <Typography variant="title">Epidemia</Typography>;
};
