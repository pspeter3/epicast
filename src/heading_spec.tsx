import * as React from "react";
import TestRenderer from "react-test-renderer";
import { Heading } from "./heading";

describe("Heading", () => {
    it("should match the snapshot", () => {
        const component = TestRenderer.create(<Heading />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
