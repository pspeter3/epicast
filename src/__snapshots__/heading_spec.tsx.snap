// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading should match the snapshot 1`] = `
<h2
  className="MuiTypography-root-1 MuiTypography-title-7"
>
  Epidemia
</h2>
`;
